packages/teams-js/test/private/chat.spec.ts

```diff
@@ -131,6 +161,16 @@ describe('chat', () => {
       return expect(chat.openChat(chatRequest)).rejects.toThrowError('The library has not yet been initialized');
     });
 
+    it('should throw error if it is not supported in runtime config', async () => {
+      const chatRequest: OpenSingleChatRequest = {
+        user: 'someUPN',
+        message: 'someMessage',
+      };
+      await utils.initializeWithContext('content');
+      utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
+      const promise = chat.openChat(chatRequest);
+      expect(promise).rejects.toThrowError(JSON.stringify({ errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM }));
```

Comment:
Maybe we can declare a variable/constant somewhere up top of the test for `JSON.stringify({ errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM })`?